dipy/tracking/utils.py

```diff
@@ -483,13 +483,19 @@ def random_seeds_from_mask(mask, seeds_count=1, seed_count_per_voxel=True,
     else:
         seeds_per_voxel = seeds_count
 
-    # Generate as many random triplets as the number of seeds needed
-    grid = np.random.random([seeds_per_voxel * num_voxels, 3])
-    # Repeat elements of 'where' so that it can be added to grid
-    where = np.repeat(where, seeds_per_voxel, axis=0)
-    seeds = where + grid - .5
+    seeds = []
+    for i in range(1, seeds_per_voxel + 1):
```

Comment:
This looks much slower than before

Reply:
It is possible but you call this function one time so I think if it's slower it's not a real problem.

Reply:
I will benchmark before and after the fix

Reply:
Yes, it is slower I check to be faster.

Reply:
So, @Garyfallidis I think I can't be more faster the slower line is to set the random seed in numpy.

Reply:
This will definitively be a bit slower than before. However, I think that, as @GuillaumeTh said, this is called only once at the beginning, to generate the seeds. In the overall processing time, this should be negligible, and has the big advantage of allowing a truly reproducible tracking (between runs on the same dataset).